javascript/node/selenium-webdriver/chrome.js

```diff
@@ -150,19 +151,23 @@ class ServiceBuilder extends chromium.ServiceBuilder {
   /**
    * @param {string=} opt_exe Path to the server executable to use. If omitted,
    *     the builder will attempt to locate the chromedriver on the current
-   *     PATH.
+   *     PATH. If the chromedriver is not available in path, selenium-manager will
+   *     download the chromedriver
    * @throws {Error} If provided executable does not exist, or the chromedriver
    *     cannot be found on the PATH.
    */
   constructor(opt_exe) {
     let exe = opt_exe || locateSynchronously()
+
     if (!exe) {
-      throw Error(
-        `The ChromeDriver could not be found on the current PATH. Please ` +
-          `download the latest version of the ChromeDriver from ` +
-          `http://chromedriver.storage.googleapis.com/index.html and ensure ` +
-          `it can be found on your PATH.`
+      console.log(
+        ` The ChromeDriver could not be found on the current PATH, trying Selenium Manager`
       )
+      exe = driverLocation(Browser.CHROME)
```

Comment:
we want to throw/catch this error so that people don't even know we tried using the Selenium Manager.